JDI.Light/JDI.Light/Factories/WebDriverFactory.cs

```diff
@@ -163,6 +163,7 @@ public void SetRunType(string runType)
                 case "remote":
                     RunType = RunType.Remote;
                     return;
+                default: break;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty 'default' clause.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5599701)